_pages/phone-screens.md

```diff
@@ -15,70 +15,114 @@ subnav:
     href: "#after-the-call"
 ---
 
-The goal of a phone screen is to quickly assess a candidate's rough fit for the position we're hiring, and to quickly screen out candidates who will obviously not work out.
+The goal of a phone screen is to assess a candidate's rough fit for the position we're hiring, and to evaluate if a candidate should be interviewed for the position.
+
+Phone screens are performed by someone on the 18F Engineering Hiring team, and often the same person performs all phone screens for a given hiring group.
 
 {% include unconscious-bias-warning.html %}
 
 ## Purpose
 
 Broadly, a phone screen accomplishes two purposes:
 
-1. **Make sure the candidate is aware of the unique nature of the job, and is happy to move forward.** Working at 18F isn't for everyone: some candidates are paid less than past positions, have to deal with more bureaucracy, and face a monumentally difficult task. If candidates aren't up for the mission and challenge, we want to find that out quickly!
+1. **Make sure the candidate is aware of the unique nature of the job, and is happy to move forward.** Working at 18f presents unique challenges of working in government, remotely, and with external partners. If candidates aren't up for the mission and challenge, we want to find that out quickly! It's also not uncommon for candidates to self-select out after the phone screen.
 
-2. **Quickly screen out candidates who obviously won't work out.** Typically this is true for candidates who don't share our values, as that's something that's hard to assess from a resume but can shake out very quickly in a conversation. Occasionally it's about technical fit: a candidate has a resume that passed review, but in talking to them it's clear they want to work on things that don't really match what we do (or the resume made it sound like they did work that they seem to not really have done).
-
-Phone screens are performed by someone on the 18F Engineering Hiring team, and often the same person performs all phone screens for a given hiring group.
+2. **Screen out candidates who would not be successful.** Candidates who don't share our values, do not match the required skills for the position, or are interested in working on things or in ways which we cannot offer.
 
-Most candidates pass the phone screen. It's a rough filter: it can't be used to really qualify a candidate, but it can be used to disqualify. It's also not uncommon for candidates to self-select out after the phone screen. This is particularly true for candidates who are financially-motivated; the GS-scale can be below what some candidates feel is appropriate.
+Phone screens are a rough filter: it can't be used to really qualify a candidate, but it can be used to disqualify.
```

Comment:
I'd moved that up as part of the first bullet - because i think it ties more to that aspect of fit. Do you think it flows better as part of the second purpose.